packages/shared/src/components/profile/UpvoterList.tsx

```diff
@@ -16,19 +16,11 @@ export function UpvoterList({
   queryResult,
   ...props
 }: UpvoterListProps): ReactElement {
-  const canFetchMore =
-    !queryResult.isLoading &&
-    !queryResult.isFetchingNextPage &&
-    queryResult.hasNextPage &&
-    queryResult.data.pages.length > 0;
-
-  const infiniteScrollRef = useFeedInfiniteScroll({
-    fetchPage: queryResult.fetchNextPage,
-    canFetchMore,
-  });
-
   return (
-    <div className="flex relative flex-col">
+    <InfiniteScrolling
```

Comment:
Perhaps we can also add a "amount" to this hook?

Reply:
Not sure where we are going to use the `amount` property.